test/e2e/common/pod_resize.go

```diff
@@ -66,6 +70,7 @@ type TestContainerInfo struct {
 	Allocations *ContainerAllocations
 	CPUPolicy   *v1.ContainerResizePolicy
 	MemPolicy   *v1.ContainerResizePolicy
+	RestartCnt	int32
```

Comment:
make update FORCE_ALL (hack/update-gofmt would fix the formatting) - let's just name it ExpectedRestartCount to be very clear this is used in verify.